data/brokers.json

```
@@ -168,5 +168,23 @@
         "costOverview": "https://www.interactivebrokers.ie/en/index.php?f=48059",
         "website": "https://www.interactivebrokers.ie/en/home.php",
         "affiliateLink": "https://ibkr.com/referral/nic106"
+    },
+    {
+        "name": "Interactive Brokers Tiered",
+        "logo": "interactive-brokers.png",
+        "product": "Tiered Pricing",
+        "etfTransactionFee": {
+            "percentage": 0.0558,
```

Misschien wel even vastleggen hoe je aan deze komt. Want dit is je eigen benadering zodat het in dit model past.

Als ik even snel kijk dan klopt dit best redelijk. Alleen voor de inleg rond de 2500-6500 zit je ietwat aan de lage kant. Maar goed, dat gaat even niet anders in deze setup.

---

JSON staat officieel volgens mij geen commentaar toe in het formaat, of is de door jullie gebruikte parser daar wat flexibeler in? Hoe/waar stel je anders voor dit te documenteren?

---

> JSON staat officieel volgens mij geen commentaar toe in het formaat, of is de door jullie gebruikte parser daar wat flexibeler in? Hoe/waar stel je anders voor dit te documenteren?

Die zou ik ff met @nicwortel checken, anders een losse readme.md ergens er bij?

---

JSON ondersteunt inderdaad geen native comments, maar ik ben het met @gerbenvl eens dat het wenselijk is om hier wel wat uitleg bij te geven. Met name zodat in de toekomst makkelijk te herleiden is of de gebruikte tarieven nog up-to-date zijn of bijgewerkt moeten worden.

Wat wel kan is het toevoegen van extra properties aan de JSON, die niet uitgelezen worden door de code maar wel wat context kunnen geven.

Technisch gezien kan zo'n property elke willekeurige key/naam hebben die nog niet in gebruik is, maar om duidelijk te maken dat het om een comment gaat die geen effect heeft op de werking van de website zou het mijn voorkeur hebben dat zo'n property een naam als `_comment` of iets dergelijks heeft.

Bijvoorbeeld:

```suggestion
            "_comment": "This is the sum of X, Y and Z",
            "percentage": 0.0558,
```